parser/src/snapshots/rustpython_parser__string__tests__fstring_parse_self_documenting_base_more.snap

```diff
@@ -23,7 +23,7 @@ expression: parse_ast
     ),
     Constant(
         ExprConstant {
-            range: 0..38,
+            range: 6..13,
```

Comment:
Same here. It seems strange that two following constants have the same range.

Reply:
We plan to remove this synthesized `ExprConstant` as well as the one above in a follow up PR